src/Backend/Modules/Pages/Domain/Page/PageVersionDataGrid.php

```diff
@@ -0,0 +1,73 @@
+<?php
+
+namespace Backend\Modules\Pages\Domain\Page;
+
+use Backend\Core\Engine\Authentication;
+use Backend\Core\Engine\DataGridDatabase;
+use Backend\Core\Engine\DataGridFunctions;
+use Backend\Core\Engine\Model;
+use Backend\Core\Language\Language;
+
+class PageVersionDataGrid extends DataGridDatabase
+{
+    public function __construct(Page $page, Status $status)
+    {
+        $whereStatement = 'p.id = :id AND p.status = :status AND p.locale = :locale';
+        $parameters =             [
```

Comment:
That's a lot of spaces before the bracket :eyes:

Reply:
fixed in 560f3fb68